scripts/sauce/sauce_connect_setup.sh

```diff
@@ -20,7 +19,15 @@ CONNECT_LOG="$LOGS_DIR/sauce-connect"
 CONNECT_STDOUT="$LOGS_DIR/sauce-connect.stdout"
 CONNECT_STDERR="$LOGS_DIR/sauce-connect.stderr"
 
-# Get Connect and start it
+# Get the appropriate URL for downloading Sauce Connect
+if [ `uname -s` = "Darwin" ]; then
```

Comment:
We do need to support Windows contributors, but I'm okay with omitting it in this PR if we file an issue for it. I tried doing `ng serve` on my home Windows machine today and that didn't work already, so there's higher-priority Windows stuff.